ezgui/src/screen_geom.rs

```diff
@@ -124,4 +132,23 @@ impl ScreenDims {
             }
         }
     }
+
+    pub fn scaled(&self, factor: f64) -> ScreenDims {
```

Comment:
I was going to suggest implementing `std::ops::Mul` instead, but actually, I think this method is unused, so remove?